daemon/src/maker_inc_connections.rs

```diff
@@ -78,6 +79,7 @@ impl Actor {
             noise_priv_key,
             heartbeat_interval,
             setup_actors: AddressMap::default(),
+            rollover_actors: AddressMap::default(),
```

Comment:
We do not yet write to this map so this can't work at the moment.

I think you'll have to introduce the equivalent of `maker_inc_connections::ConfirmOrder` for rollover and send the address across.

Reply:
```
        self.rollover_actors
            .insert(proposal.order_id, rollover_actor_addr);
```

I add the rollover actor to this map in `maker_cfd::handle_propose_rollover`

Reply:
But here we are talking about the `maker_inc_connections::Actor`! :)

This map is (unfortunately) in two places and we need to populate both of them.

Reply:
Checkout how `maker_inc_connections::settlement::Response` is done.

Reply:
> But here we are talking about the `maker_inc_connections::Actor`! :)
> 
> This map is (unfortunately) in two places and we need to populate both of them.

Ahh i see the problem now